lib/pbench/agent/base.py

```diff
@@ -97,8 +97,7 @@ def execute(self):
 
     def get_path(self, path):
         """Converts a string path into a pathlib object"""
-        if path:
-            if not isinstance(path, pathlib.PurePath):
-                return pathlib.Path(path)
+        if not isinstance(path, pathlib.PurePath):
+            return pathlib.Path(path)
```

Comment:
But if `path` is `None`, `pathlib.PurePath` is going to be unhappy. Maybe that's OK if we know it can never be None; but you were checking explicitly before.

Reply:
Yeah fixed